images/installer-origin-release/Dockerfile.ci

```diff
@@ -0,0 +1,11 @@
+# This Dockerfile is used as a CI job to build smoke tests for OpenShift installer
+
+FROM  quay.io/coreos/origin-release-oc:bazel
+WORKDIR /go/src/github.com/openshift/installer
+COPY . .
+
+ENV USER="bazel"
+ENV HOME="/tmp"
+
+RUN mkdir /output && \
+    bazel --output_base=/output/bazel-build build smoke_tests
```

Comment:
This puts a lot of stuff in `/output`.  And actually, I've just realized that the point of this Dockerfile is to add the smoke-test binary, not to drop the smoke test binary into a mounted volume :man_facepalming:.  So I think we probably want to follow the `yum` pattern and clean up after ourselves in the same command to keep the image layer small.  How about:

```Dockerfile
RUN bazel --output_base=/tmp/bazel-output build smoke_tests && \
  cp bazel-bin/tests/smoke/linux_amd64_stripped/go_default_test /usr/bin/smoke && \
  bazel clean
```

or something?

Reply:
testing with that...